keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -291,80 +686,149 @@ No, the default behavior remains the same.
+
+Usage: Administrators and SREs can use this metric to monitor and alert on StatefulSet configurations that violate the maxUnavailable constraint. This metric is particularly useful for ensuring that the rolling updates are proceeding as expected and that no unexpected unavailability is occurring. This metric can be aggregated across multiple clusters for large-scale monitoring.
+
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
-Will be tested when graduating to Beta.
 
+No, but it will be tested manually before merging the PR.
 
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
 No
 
 ### Monitoring Requirements
 
 ###### How can an operator determine if the feature is in use by workloads?
+
 If their StatefulSet rollingUpdate section has the field maxUnavailable specified with
-a value different than 1.
-The below command should show maxUnavailable value:
+a value different than 1. While in alpha and beta, the feature-gate needs to be enabled.
+
+The command bellow should show the maxUnavailable value:
 ```
 kubectl get statefulsets -o yaml | grep maxUnavailable
 ```
 
 ###### How can someone using this feature know that it is working for their instance?
-TODO when we reach Beta
+
+With feature enabled, set the `maxUnavailable` great than 1, and pay attention to the rolling update pods at a time,
+it should equal to the `maxUnavailable`.
+Or refer to the `rolling-update-duration-seconds`, it can give some indicators generally. Like when setting the `maxUnavailable`
+greater than 1, then the duration should descend.
 
 ###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
 
+<!--
+This is your opportunity to define what "normal" quality of service looks like
+for a feature.
+
+It's impossible to provide comprehensive guidance, but at the very
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+Less than 0.5% of rolling updates should experience unavailability violations where the number of unavailable pods exceeds the maxUnavailable limit over a rolling 30-day window.
```

Comment:
If we are relying on `statefulset_unavailability_violation` this could also include instances where pods are failing for other reasons and not necessarily determine if this feature is operator correctly, correct?